crates/op-rbuilder/src/tx.rs

```diff
@@ -18,7 +18,11 @@ pub trait FBPoolTransaction: MaybeRevertingTransaction + OpPooledTx {}
 #[derive(Clone, Debug)]
 pub struct FBPooledTransaction {
     pub inner: OpPooledTransaction,
-    pub exclude_reverting_txs: bool,
+
+    /// reverted hashes for the transaction. If the transaction is a bundle,
+    /// this is the list of hashes of the transactions that reverted. If the
+    /// transaction is not a bundle, this is `None`.
+    pub reverted_hashes: Option<Vec<B256>>,
```

Comment:
opinionated nit: should we just treat an empty vec as nothing? This might add an unnecessary `Option` check burden on the code user.